back/flag-service/src/flag/FlagController.ts

```diff
@@ -51,7 +53,15 @@ export class FlagController {
     }
   }
 
+  @Get('pixel')
+  async getUserPixel(
+      @UserId() userId: string
```

Comment:
We should enforce commas on these.
Please add the eslint rule `'comma-dangle': ['error', 'always']`.